hedera-mirror-rest/topicmessage.js

```diff
@@ -101,14 +101,23 @@ const getMessageByTopicAndSequenceRequest = async (req, res) => {
   const seqNum = req.params.sequenceNumber;
   validateGetSequenceMessageParams(topicIdStr, seqNum);
   const topicId = EntityId.parse(topicIdStr);
-
-  // handle topic stated as x.y.z vs z e.g. topic 7 vs topic 0.0.7.
-  const pgSqlQuery = `select *
-                      from ${TopicMessage.tableName}
-                      where ${TopicMessage.TOPIC_ID} = $1
-                        and ${TopicMessage.SEQUENCE_NUMBER} = $2
-                      limit 1`;
-  const pgSqlParams = [topicId.getEncodedId(), seqNum];
+  let pgSqlQuery;
+  let pgSqlParams;
+  if (config.query.topicMessageLookup) {
+    const query = await getQueryWithTopicMessageLookupTimestamps(topicId, seqNum);
+    if (query === null) {
+      return null;
+    }
+    pgSqlQuery = query.pgSqlQuery;
+    pgSqlParams = query.pgSqlParams;
```

Comment:
prefer object destructuring syntax

```suggestion
    {pgSqlQuery,  pgSqlParams} = query;
```